src/js/component/Api.js

```diff
@@ -0,0 +1,33 @@
+import {Errors} from './Errors';
+export class Api {
+    constructor(BASE_URL) {
+      this.BASE_URL = BASE_URL
+    }
+       
+   async request(url, method, body) {
+      const param = {
+          method,
+          body: JSON.stringify(body)
+      }
+      try {
+        return await fetch(this.BASE_URL + url , param).then(res => {
+          console.log(`Status: ${res.status} Url:${res.url}`)
```

Comment:
Lets move it to proxy

Reply:
I understood little late
Console.log shifted to proxy